src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -1358,6 +1358,10 @@
 			"economy.new_expand.price_claim_townblock_increase",
 			"1.0",
 			"# How much every additionally claimed townblock increases in cost. Set to 1 to deactivate this. 1.3 means +30% to every bonus claim block cost."),
+	ECO_MAX_PRICE_CLAIM_TOWNBLOCK(
+			"economy.new_expand.max_price_claim_townblock",
+			"-1.0",

```

Comment:
Please add a `"",` below the `"-1.0",` line so that the new whitespacing is maintained.